src/dtls13.c

```diff
@@ -1447,40 +1449,41 @@ static int _Dtls13HandshakeRecv(WOLFSSL* ssl, byte* input, word32 size,
 #endif /* WOLFSSL_DEBUG_TLS */
 
         /* ignore the message */
-        *processedSize = idx + frag_length;
-
-        *processedSize += ssl->keys.padSz;
+        *processedSize = idx + fragLength + ssl->keys.padSz;
 
         return 0;
     }
 
-    isFirst = frag_off == 0;
-    isComplete = isFirst && frag_length == message_length;
+    isFirst = fragOff == 0;
+    isComplete = isFirst && fragLength == messageLength;
+
+    isAsyncEnabled = 0;
+#ifdef WOLFSSL_ASYNC_CRYPT
+    isAsyncEnabled = 1;
```

Comment:
Seems like there must be a better way? Is ssl->error == WC_PENDING_E yet? In the async case this looks like = 0 then = 1 which might trigger a static analyzer warning.